Dockerfile

```diff
@@ -8,13 +8,29 @@ ARG DEBIAN_FRONTEND=noninteractive
 ARG VPNKIT_VERSION=0.4.0
 ARG DOCKER_BUILDTAGS="apparmor seccomp selinux"
 ARG GOLANG_IMAGE="golang:${GO_VERSION}-buster"
+ARG BASE_IMAGE="debian:buster"
```

Comment:
Did you want to go with `debian:buster-slim` to exclude man pages/docs and get smaller image sizes?

Reply:
Already pulling in buildpack-deps:buster for the golang image. 🤷 
Might be nice to just have a `FROM scratch` image with go in it in that case.